e2e/plain-views/expected-output.diff

```diff
@@ -4,7 +4,7 @@
 1) src/short_open_tag_with_import_name.php:0
 
     ---------- begin diff ----------
-@@ @@
+@@ -1,10 +1,12 @@
  <?php
 +
```

Comment:
This somehow change the original expectation

https://github.com/rectorphp/rector-src/blob/5c1be93b80c3f7bfd8bd158ba0a55501a7acaefc/e2e/plain-views/expected-output.diff#L10

Reply:
First stmt detection maybe changed, you may need to debug on 

https://github.com/rectorphp/rector-src/blob/5c1be93b80c3f7bfd8bd158ba0a55501a7acaefc/packages/PostRector/Rector/NameImportingPostRector.php#L58-L61